modules/Lending.py

```diff
@@ -114,7 +119,9 @@ def notify_new_loans(sleep_time):
     try:
         new_provided = api.return_active_loans()['provided']
         if loans_provided:
-            get_id_set = lambda loans: set([x['id'] for x in loans]) # lambda to return a set of ids from the api result
+            # function to return a set of ids from the api result
+            # get_id_set = lambda loans: set([x['id'] for x in loans])
+            def get_id_set(loans): return set([x['id'] for x in loans])
```

Comment:
The original is more pythonic imo. Do you feel it's hard for people to read / understand?

Reply:
Now I remember and found it again: 
https://www.python.org/dev/peps/pep-0008/#programming-recommendations
"Always use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier"